packages/taquito-local-forging/test/taquito-local-forging.spec.ts

```diff
@@ -28,9 +31,9 @@ describe('Forge and parse operations default protocol', () => {
     });
   });
 
-  describe('Forge and parse operations kathmandu protocol', () => {
+  describe('Forge and parse operations lima protocol', () => {
     const localForger = new LocalForger();
```

Comment:
LocalForger should be instantiated with the limanet protocol hash, or it will use Kathmandu by default.. I don't understand with the tests are passing using this protocol hash.